@here/harp-mapview-decoder/test/TileGeometryLoaderTest.ts

```diff
@@ -25,6 +24,8 @@ import * as sinon from "sinon";
 
 chai.use(chaiAsPromised);
 const { expect } = chai;
+chai.should();
```

Comment:
This is probably not needed?

Reply:
It's part of the setup to be able to use "should" with "chai-as-promised". Otherwise the chai-as-promised extensions don't work with should.

Reply:
ah ok, thanks for clarifiing